src/js/utils/ValidatorUtil.js

```diff
@@ -34,22 +34,22 @@ var ValidatorUtil = {
   },
 
   isInteger: function (value) {
-    const number = parseFloat(value);
-
-    return Number.isInteger(number);
+    return ValidatorUtil.isNumber(value) &&
+      Number.isInteger(parseFloat(value));
```

Comment:
We probably should rename these methods as they intentional use `parseFloat` to also handle number like  strings. How do you feel about `isNumericValue`, `isIntegerValue` , and `isNumericValueInRange`?